livekit-android-camerax/README.md

```diff
@@ -19,7 +19,10 @@ See our [release page](https://github.com/livekit/client-sdk-android/releases) f
 ### Register the CameraProvider
 
 ```
-CameraXHelper.createCameraProvider(lifecycleOwner).let {
```

Comment:
This one's unneeded since you add it specifically for the image capture section below.